.stylelintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+  "rules": {
+    "property-disallowed-list": [
+      "margin-left",
+      "margin-right",
+      "padding-left",
+      "padding-right",
+      "border-left",
+      "border-right",
+      "border-inline",
```

Comment:
not used often, but what's wrong with this one?

Reply:
The `border-inline` & `border-inline-color` & `border-inline-style` (same for block) don't have adequate level of browser support.

You should use: `boder-inline-end` & `border-inline-start` if it's only one side. If you want a border all around or on both sides you can use the `border` shorthand.

Reply:
gotcha